### ISSUE TITLE
Cannot build on Windows

### ISSUE THREAD

**Issue by laszlocsomor** [bazelbuild/rules_jvm_external#364]
On Bazel CI, at 0a7cc6a0b6764232a0ddd31ad87b489e1d47b166, with Bazel 2.0.0 (and also at HEAD: https://github.com/bazelbuild/bazel/commit/f5b9d930a93fadd40295ffe38dec2bf80569e38b) on Windows, we get build failures.

The look the same as https://github.com/bazelbuild/examples/issues/146

Log: https://buildkite.com/bazel/rules-jvm-external-examples/builds/1013#28fc946e-19ff-4c4b-86af-a34f00d670e3

```

(00:15:13) INFO: From Dexing external/maven/_dx/org_ccil_cowan_tagsoup_tagsoup/stamped_tagsoup-1.2.jar_desugared.jar with applicable dexopts []:
--
  | warning: Ignoring InnerClasses attribute for an anonymous inner class
  | (org.ccil.cowan.tagsoup.Parser$1) that doesn't come with an
  | associated EnclosingMethod attribute. This class was probably produced by a
  | compiler that did not target the modern .class file format. The recommended
  | solution is to recompile the class from source, using an up-to-date compiler
  | and without specifying any "-target" type options. The consequence of ignoring
  | this warning is that reflective operations on this class will incorrectly
  | indicate that it is *not* an inner class.
  | (00:15:13) ERROR: D:/b/hmndq6yn/external/maven/BUILD:173:1: Couldn't build file external/maven/com_android_support_appcompat_v7_symbols/symbols.zip: Compiling Android resources for @maven//:com_android_support_appcompat_v7 failed (Exit 1)
  | Jan 22, 2020 12:15:13 AM java.util.stream.ForEachOps$ForEachOp$OfRef accept
  | SEVERE: Error during Compiling bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-hdpi-v4\abc_btn_switch_to_on_mtrl_00001.9.png:
  | Command: external\androidsdk\build-tools\29.0.2\aapt2.exe\
  | compile\
  | -v\
  | --legacy\
  | -o\
  | D:\temp\android_resources_tmp15986696223201690866\compiled\bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-hdpi-v4\abc_btn_switch_to_on_mtrl_00001.9.png
  | Output:
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-hdpi-v4\abc_btn_switch_to_on_mtrl_00001.9.png: note: compiling PNG.
  |  
  | Jan 22, 2020 12:15:13 AM java.util.stream.ForEachOps$ForEachOp$OfRef accept
  | SEVERE: Error during Compiling bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_ab_share_pack_mtrl_alpha.9.png:
  | Command: external\androidsdk\build-tools\29.0.2\aapt2.exe\
  | compile\
  | -v\
  | --legacy\
  | -o\
  | D:\temp\android_resources_tmp15986696223201690866\compiled\bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_ab_share_pack_mtrl_alpha.9.png
  | Output:
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_ab_share_pack_mtrl_alpha.9.png: note: compiling PNG.
  |  
  | Jan 22, 2020 12:15:13 AM java.util.stream.ForEachOps$ForEachOp$OfRef accept
  | SEVERE: Error during Compiling bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_list_divider_mtrl_alpha.9.png:
  | Command: external\androidsdk\build-tools\29.0.2\aapt2.exe\
  | compile\
  | -v\
  | --legacy\
  | -o\
  | D:\temp\android_resources_tmp15986696223201690866\compiled\bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_list_divider_mtrl_alpha.9.png
  | Output:
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_list_divider_mtrl_alpha.9.png: note: compiling PNG.
  |  
  | Jan 22, 2020 12:15:13 AM java.util.stream.ForEachOps$ForEachOp$OfRef accept
  | SEVERE: Error during Compiling bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_spinner_mtrl_am_alpha.9.png:
  | Command: external\androidsdk\build-tools\29.0.2\aapt2.exe\
  | compile\
  | -v\
  | --legacy\
  | -o\
  | D:\temp\android_resources_tmp15986696223201690866\compiled\bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_spinner_mtrl_am_alpha.9.png
  | Output:
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_spinner_mtrl_am_alpha.9.png: note: compiling PNG.
  |  
  | Exception in thread "main" com.google.devtools.build.android.aapt2.ResourceCompiler$CompileError
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileError.of(ResourceCompiler.java:110)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompilingVisitor.getCompiledArtifacts(ResourceCompiler.java:422)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler.getCompiledArtifacts(ResourceCompiler.java:456)
  | at com.google.devtools.build.android.UnvalidatedAndroidData.compile(UnvalidatedAndroidData.java:103)
  | at com.google.devtools.build.android.CompileLibraryResourcesAction.main(CompileLibraryResourcesAction.java:140)
  | at com.google.devtools.build.android.ResourceProcessorBusyBox$Tool$10.call(ResourceProcessorBusyBox.java:120)
  | at com.google.devtools.build.android.ResourceProcessorBusyBox.processRequest(ResourceProcessorBusyBox.java:237)
  | at com.google.devtools.build.android.ResourceProcessorBusyBox.main(ResourceProcessorBusyBox.java:198)
  | Suppressed: java.lang.RuntimeException: Error during Compiling bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-hdpi-v4\abc_btn_switch_to_on_mtrl_00001.9.png:
  | Command: external\androidsdk\build-tools\29.0.2\aapt2.exe\
  | compile\
  | -v\
  | --legacy\
  | -o\
  | D:\temp\android_resources_tmp15986696223201690866\compiled\bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-hdpi-v4\abc_btn_switch_to_on_mtrl_00001.9.png
  | Output:
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-hdpi-v4\abc_btn_switch_to_on_mtrl_00001.9.png: note: compiling PNG.
  |  
  | at com.google.devtools.build.android.CommandHelper.execute(CommandHelper.java:42)
  | at com.google.devtools.build.android.AaptCommandBuilder.execute(AaptCommandBuilder.java:297)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.compile(ResourceCompiler.java:226)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.call(ResourceCompiler.java:173)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.call(ResourceCompiler.java:120)
  | at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
  | at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
  | at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
  | at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
  | at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
  | at java.base/java.lang.Thread.run(Thread.java:834)
  | Suppressed: java.lang.RuntimeException: Error during Compiling bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_ab_share_pack_mtrl_alpha.9.png:
  | Command: external\androidsdk\build-tools\29.0.2\aapt2.exe\
  | compile\
  | -v\
  | --legacy\
  | -o\
  | D:\temp\android_resources_tmp15986696223201690866\compiled\bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_ab_share_pack_mtrl_alpha.9.png
  | Output:
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_ab_share_pack_mtrl_alpha.9.png: note: compiling PNG.
  |  
  | at com.google.devtools.build.android.CommandHelper.execute(CommandHelper.java:42)
  | at com.google.devtools.build.android.AaptCommandBuilder.execute(AaptCommandBuilder.java:297)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.compile(ResourceCompiler.java:226)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.call(ResourceCompiler.java:173)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.call(ResourceCompiler.java:120)
  | at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
  | at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
  | at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
  | at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
  | at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
  | at java.base/java.lang.Thread.run(Thread.java:834)
  | Suppressed: java.lang.RuntimeException: Error during Compiling bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_list_divider_mtrl_alpha.9.png:
  | Command: external\androidsdk\build-tools\29.0.2\aapt2.exe\
  | compile\
  | -v\
  | --legacy\
  | -o\
  | D:\temp\android_resources_tmp15986696223201690866\compiled\bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_list_divider_mtrl_alpha.9.png
  | Output:
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_list_divider_mtrl_alpha.9.png: note: compiling PNG.
  |  
  | at com.google.devtools.build.android.CommandHelper.execute(CommandHelper.java:42)
  | at com.google.devtools.build.android.AaptCommandBuilder.execute(AaptCommandBuilder.java:297)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.compile(ResourceCompiler.java:226)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.call(ResourceCompiler.java:173)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.call(ResourceCompiler.java:120)
  | at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
  | at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
  | at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
  | at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
  | at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
  | at java.base/java.lang.Thread.run(Thread.java:834)
  | Suppressed: java.lang.RuntimeException: Error during Compiling bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_spinner_mtrl_am_alpha.9.png:
  | Command: external\androidsdk\build-tools\29.0.2\aapt2.exe\
  | compile\
  | -v\
  | --legacy\
  | -o\
  | D:\temp\android_resources_tmp15986696223201690866\compiled\bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_spinner_mtrl_am_alpha.9.png
  | Output:
  | bazel-out\android-armeabi-v7a-fastbuild\bin\external\maven\_aar\unzipped\resources\com_android_support_appcompat_v7\res\drawable-xhdpi-v4\abc_spinner_mtrl_am_alpha.9.png: note: compiling PNG.
  |  
  | at com.google.devtools.build.android.CommandHelper.execute(CommandHelper.java:42)
  | at com.google.devtools.build.android.AaptCommandBuilder.execute(AaptCommandBuilder.java:297)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.compile(ResourceCompiler.java:226)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.call(ResourceCompiler.java:173)
  | at com.google.devtools.build.android.aapt2.ResourceCompiler$CompileTask.call(ResourceCompiler.java:120)
  | at com.google.common.util.concurrent.TrustedListenableFutureTask$TrustedFutureInterruptibleTask.runInterruptibly(TrustedListenableFutureTask.java:125)
  | at com.google.common.util.concurrent.InterruptibleTask.run(InterruptibleTask.java:57)
  | at com.google.common.util.concurrent.TrustedListenableFutureTask.run(TrustedListenableFutureTask.java:78)
  | at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
  | at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
  | at java.base/java.lang.Thread.run(Thread.java:834)


```

**Comment by laurentlb** [comment]
@jin

**Comment by jin** [comment]
Thanks for ping. Looks like an issue from the Android rules (specifically resource processor busybox), which is not related to rules_jvm_external. Will take a look.

**Comment by laurentlb** [comment]
For the record, I've got failures with Bazel 2.1rc: https://buildkite.com/bazel/bazel-at-head-plus-downstream/builds/1373

I'd like to know if it's a release blocker or not.

**Comment by jin** [comment]
@laurentlb this is not a 2.1 issue. I can reproduce this with Bazel 1.0.0. So this is not a release blocker.

**Comment by jin** [comment]
Disabling all Android build pipelines on Windows for the time being.

**Comment by meteorcloudy** [comment]
@jin Looks like our android rule is not compatible with build tool 29.0.3.
I'll ping the sdk version to 28.0.2 to make our downstream green.

**Comment by jin** [comment]
Thank you, @meteorcloudy
